src/xrCore/_math.cpp

```diff
@@ -15,9 +15,22 @@
 //?
 #endif
 
-#elif defined(LINUX)
+#elif defined(LINUX) || defined(FREEBSD)
 #include <x86intrin.h> // __rdtsc
+#ifdef LINUX
 #include <fpu_control.h>
+#else
```

Comment:
```suggestion
#elif defined(FREEBSD)
```
It makes code cleaner